Info-Orbs/lib/globalTime/globalTime.cpp

```diff
@@ -25,8 +25,9 @@ GlobalTime *GlobalTime::getInstance() {
 
 void GlobalTime::updateTime() {
     if (millis() - m_updateTimer > m_oneSecond) {
-        if (m_timeZoneOffset != 0) {
-            m_timeClient->setTimeOffset(3600 * m_timeZoneOffset);
+        if (m_timeZoneOffset == 1000) {
```

Comment:
maybe a default value of "-1" or a constant to indicate unset?

Reply:
ahh, i thought -1 does not work because timezone can also be negative, but the response is already multiplied by 3600 so it should work